org.kde.rkward.json

```
@@ -0,0 +1,74 @@
+{
+    "id": "org.kde.rkward",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "rkward",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name" : "R",
+            "buildsystem" : "autotools",
+            "config-opts" : [ "--enable-R-shlib=yes" ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://cloud.r-project.org/src/base/R-4/R-4.1.3.tar.gz",
+                    "sha256" : "15ff5b333c61094060b2a52e9c1d8ec55cc42dd029e39ca22abdaa909526fed6",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 4150,
+                        "stable-only": true,
+                        "url-template": "https://cloud.r-project.org/src/base/R-4/R-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "kate",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/22.04.0/src/kate-22.04.0.tar.xz",
+                    "sha256": "ca90ad237113a846715ee6b6bfb6aff0d2f9031eed15df412632411ce93fc8cb",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 236337,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/release-service/$version/src/kate-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "buildsystem": "cmake",
```

```suggestion
            "buildsystem": "cmake-ninja",
```

(unless it doesn't work)